go/libraries/doltcore/sqle/dsess/session.go

```diff
@@ -100,6 +100,14 @@ func init() {
 			Type:              sql.NewSystemBoolType(ForceTransactionCommit),
 			Default:           int8(0),
 		},
+		{
+			Name:              doltdb.ReplicateHeadsStrategy,
+			Scope:             sql.SystemVariableScope_Session,
+			Dynamic:           true,
+			SetVarHintApplies: false,
+			Type:              sql.NewSystemStringType(doltdb.ReplicateHeadsStrategy),
+			Default:           "one",
```

Comment:
Default should be many, only replicating one head is an advanced setup and probably not what you want

And the configuration should be "all" or "the ones I name specifically"

Reply:
I think you want two system vars:

`replicate_all_heads` (boolean)
`replicate_heads` (comma separated list of heads)